packages/radix-vue/src/Calendar/CalendarPrev.vue

```diff
@@ -1,10 +1,15 @@
 <script lang="ts">
 import type { PrimitiveProps } from '@/Primitive'
 import type { CalendarIncrement } from '@/shared/date'
+import type { DateValue } from '@internationalized/date'
 
 export interface CalendarPrevProps extends PrimitiveProps {
-/** The calendar unit to go back */
+/** The calendar unit to go back 
+* @deprecated Use `prevPage` instead
+*/
   step?: CalendarIncrement
+/** The function to be used for the prev page. Overwrites the `prevPage` function set on the `CalendarRoot`. */
+  prevPage?: (placeholder: DateValue) => DateValue
```

Comment:
why do we need `prevPage` if we already provide the props in `CalendarRoot`?

Reply:
For the case when you want multiple `CalendarNext`/`CalendarPrev` components inside the `CalendarRoot` (e.g. the default `CalendarNext`/`CalendarPrev` increments by `month`, the others by `year`) like in the `CalendarWithYearIncrements` story.